pages/nettest.js

```diff
@@ -191,7 +188,9 @@ export default class extends React.Component {
           </Link>
 
           <Box mt={5}>
-            <Code>{userAgent}</Code>
```

Comment:
`Code` component is deprecated from rebass, that's why had to use `Box` component with `as="pre"`